sql/moz-fx-data-shared-prod/firefox_ios/retention/view.sql

```diff
@@ -0,0 +1,20 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.firefox_ios.retention`
+AS
+SELECT
+  *,
+  CASE
+    WHEN first_seen_date = metric_date
+      THEN 'new_client'
+    -- TODO: Should we rename "repeat_client" to something different to avoid a mix up
```

Comment:
@bani any thoughts here?

Reply:
mix up with what? I was trying to use terminology consistent with the funnel, since someone who returns between day 2-27 goes into the repeat rate there.